Add 1.1.1 issue template and input manifests (#1275)
opensearch-project/opensearch-build: .github/meta_issue_template/release-1.1.1-plugin-issues-min.md

> This is a component issue for release 1.1.1.

Reviewer: What is the usage of this `-min` vs the other issue?

Author: It is only used for OpenSearch and Dashboards core repo.
Since they do not need any changes they also do not need any specific tests.

Reviewer: Can we adapt the single template to cover this case in the with a "What to do if you have no changes section".  This second template requires more work by the release manager that it would be better to avoid.

Author: Fixed.

Author: ```
- [ ] If including changes in this release, increment the version on 1.1 branch to `1.1.1` for Min/Core, and `1.1.1.0` for components. Otherwise, keep the version number unchanged for both.
```

Coming from [release issue 1.1.1](https://github.com/opensearch-project/opensearch-build/issues/870), release version 1.1.1. Please follow the following checklist.

<details><summary>How to use this component issue</summary>
<p>

> ## This Component Issue

Reviewer: Can we rename the these files to `components`?  Its the language used throughout the release process.  Plugin is a kind of component, but not all components are plugins like common-util

Author: Will do.

This component issue captures the state of the OpenSearch release, on component/plugin level, its assignee is responsible for driving the release of the component.  Please contact them or @mention them on this issue for help. 

## Release Steps
There are several steps to the release process, components that are behind present risk to the release.  Component owners resolve tasks on this ticket to communicate with the overall release owner.

Steps have completion dates for coordinating efforts between the components of a release; components can start as soon as they are ready far in advance of a future release.

You can find all the corresponding dates of each step in the release issue above.

</p>
</details>

### You can find all the date in above issue 

> ### (We make changes to both OpenSearch and OpenSearch-Dashboards for 1.1.1)

Reviewer: Can we remove this line - if we have nodes we should integrate them into the how to use section?  If we need teams to take different action then we should call it out in the task list below

Author: Sure.

### Preparation

- [ ] Assign this issue to a release owner.
- [ ] All the tasks in this issue have been reviewed by the release owner.
- [ ] Create, update, triage and label all features and issues targeted for this release with v1.1.1.

### CI/CD

> - [ ] If including changes in this release, increment plugin version on 1.1 branch to `1.1.1`, otherwise stay on `1.1.0`.

Reviewer: ```suggestion
- [ ] If including changes in this release, increment the version on 1.1 branch to `1.1.1`, otherwise stay on `1.1.0`.
```

Author: Fixed.

> - [ ] If including changes in this release, increment plugin version on 1.1 branch to `1.1.1`, otherwise stay on `1.1.0`.

Reviewer: Rather than add annotations for if/else in inline, what do you think about a section "What should I do if my plugin isn't making any changes?" that lays out exactly what to checkout off, including this line item.  This was part of our retro feedback on 1.2.0

Author: Good idea.

- [ ] All code changes for 1.1.1 are complete.
- [ ] Ensure working and passing CI.

### Pre-Release

> - [ ] Merge any changes to 1.1 you need for the patch

Reviewer: `All code changes for 1.1.1 are complete.` isn't this line the same as that one?

Author: This is a second confirmation just to make sure component owners have all the changes ready.

Reviewer: Rather than `merge` what do you think about making it 'confirmation' oriented?
```suggestion
- [ ] Confirm that all changes for 1.1.1 have been merged.
```

Reviewer: [Blocking] I think this instruction is confusing for teams, please update.

Author: Fixed.

> - [ ] Complete integration and sanity tests, and update results in the comment, contact corresponding assigner in meta issue above if needed.

Reviewer: Encouraging direct contact to the release owner isn't going to scale, making sure this process can be followed without any direct contact.  A link to an example would go along way here to make sure folks are following through consistently would be great.

```suggestion
- [ ] Complete integration and sanity tests, and update results in the comment, [example](link).
```

Author: I totally get that but at the moment after viewing through several issues I cannot find a good example, as component owners would just check CI results or do manual testing.

Would you think adding my side of the integTest post would help?
https://github.com/opensearch-project/opensearch-build/issues/1118

Author: Anyway I will change it with a link to my issue, and extend from there later on.
Thanks.

Reviewer: Adding a link to a `RELEASE_PROCESS.MD#HowToPublishTestResults` that we can fill in later works too.

Author: Where is this file exist on the repo?
RELEASE_PROCESS.MD#HowToPublishTestResults

Reviewer: It doesn't exist yet, but it's a great fast follow

Author: Got it.

> - [ ] Fix bugs that you find during the integration test, wait for a new build and test again.

Reviewer: Great callout, this exposes an issue, how do they wait and test again?  every day, every hour?  Or should they get a notice about a release candidate build somehow?  We used this issue, https://github.com/opensearch-project/opensearch-build/issues/954 for when we had to 'change the plan' on everyone, maybe we could cut a ticket for 'testing has completed against the RC build? in a similar way'

Here is what I would recommend, lets keep 'Find/fix bugs using latest docker image', and lets add a new line for 'completed release candidate testing build #TBD`, then while slightly painful, we could add a new line for each additional RC build that arrives with the link on where to get it.

Author: Added, thanks.
